### DISCUSSION TITLE
[@types/cli-progress] Add new functions & options

### DISCUSSION THREAD

**Discussion by longnguyen2004** [DefinitelyTyped/DefinitelyTyped#60161] [Issues with a @types package] [answered]
New `Options` prop (https://github.com/npkgz/cli-progress/commit/92ea6d3953faa7023ff8427d7a636de7aecfda71, https://github.com/npkgz/cli-progress/commit/17b9c642da9efed4ac4a226a6a3fe539c6d89d76):
```ts
interface Options {
    gracefulExit?: boolean | undefined, 
    progressCalculationRelative?: boolean | undefined
}
```

`MultiBar::create()` now accepts 1 extra parameter (https://github.com/npkgz/cli-progress/commit/31692f2610e58fa40069d69ec3da329d431076d3):
```ts
class MultiBar extends EventEmitter {
    create(total: number, startValue: number, payload?: any, barOptions?: Options)
}
```

New function: `MultiBar::log()` (https://github.com/npkgz/cli-progress/commit/c1b9375cf3c4714b97008c33ade27de719104590):
```ts
class MultiBar extends EventEmitter {
   log(message: string)
}
```

[a comment by an automated account is not shown]

**Comment by longnguyen2004** [comment]
Bump :)

**Comment by peterblazejewicz** [comment]
I think there is a PR to address those shortcomings:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60191

**Comment by longnguyen2004** [comment, marked as answer]
Closing because #60191 is merged, thanks for everyone's effort!
